src/networks.json

```diff
@@ -580,6 +580,20 @@
     "explorer": "https://polygonscan.com",
     "start": 9834491,
     "logo": "ipfs://QmaGokGqgjknfa4xnXKnnwC5ZyXzUjQ7p6KEe4D8G5uFFE"
+  },  
+  "188": {
+    "key": "188",
+    "name": "Bytom Sidechain",
+    "shortName": "BMC",
+    "chainId": 188,
+    "network": "mainnet",
+    "multicall": "0xDA09528B093246eC70139b657d3B7A3bd5F4C859",
```

Comment:
Is there a way to check contract code here https://bmc.blockmeta.com/address/0xDA09528B093246eC70139b657d3B7A3bd5F4C859

Reply:
yes

Reply:
@MartinBTM I mean verify and publish the contract code to the explorer

Reply:
hi，we have push the code to explorer, please check